Sources/Controllers/Panels/OAMapPanelViewController.h

```diff
@@ -156,14 +156,15 @@
                 leftInset:(float)leftInset
                  animated:(BOOL)animated;
 
-- (void) displayAreaOnMap:(CLLocationCoordinate2D)topLeft
-              bottomRight:(CLLocationCoordinate2D)bottomRight
-                     zoom:(float)zoom
-               screenBBox:(CGSize)screenBBox
-              bottomInset:(float)bottomInset
-                leftInset:(float)leftInset
-                 topInset:(float)topInset
-                 animated:(BOOL)animated;
+- (void)displayAreaOnMap:(CLLocationCoordinate2D)topLeft
```

Comment:
please create a separate method without maxzoom parameter and pass 22 by default